content/docs/command-reference/studio/index.md

```diff
@@ -0,0 +1,37 @@
+# studio
+
+Set of commands to authorize DVC with Studio and save the token to global [DVC
+configuration]: [login](/doc/command-reference/studio/login),
+[logout](/doc/command-reference/studio/logout),
+[token](/doc/command-reference/studio/token),
+
+[dvc configuration]:
+  /doc/user-guide/project-structure/configuration#config-file-locations
+
+## Synopsis
+
+```usage
+usage: dvc studio [-h] [-q | -v] {login,logout,token} ...
+
+positional arguments:
+  {login,logout,token}  Use `dvc studio CMD --help` to display command-specific help.
+    login               Authenticate DVC with Studio host
+    logout              Logout user from Studio
+    token               View the token dvc uses to contact Studio
+
+options:
```

Comment:
Style: we don't include option descriptions in the usage section, probably we also don't include positional arg descriptions